example/src/App.tsx

```diff
@@ -195,6 +204,12 @@ const App = () => {
           );
         })}
       </Box>
+      <Box>
+        <Typography>
+          Etherspot Smart Wallet Addresses:
+        </Typography>
+        {currentSmartWalletAddresses.map((smartWalletAddress) => <Paper key={`swa-${smartWalletAddress.chainId}`} sx={{p: 1}} variant="outlined"><Typography>Chain ID: {smartWalletAddress.chainId}<br />Address: {smartWalletAddress.address}</Typography></Paper>)}
+      </Box>
```

Comment:
if u can break it down into new lines 🙏 
```suggestion
        {currentSmartWalletAddresses.map((smartWalletAddress) => <Paper key={`swa-${smartWalletAddress.chainId}`} sx={{p: 1}} variant="outlined"><Typography>Chain ID: {smartWalletAddress.chainId}<br />Address: {smartWalletAddress.address}</Typography></Paper>)}
      </Box>
```

Reply:
Done